.prettierignore

```diff
@@ -1,3 +1,4 @@
 **/bin/**
 **/dist/**
 **/node_modules/**
+**/bower_components/**
```

Comment:
Should be added to our global [.prettierignore](https://github.com/wireapp/wire-web-packages/blob/master/.prettierignore).

Reply:
But this is the global `.prettierignore` file.